src/lib/Form/Data/Role/RoleCopyData.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Form\Data\Role;
+
+use eZ\Publish\API\Repository\Values\User\Role;
+
+class RoleCopyData
+{
+    /** @var Role */
+    private $clonedRole;
```

Comment:
I'd call it `$copiedRole` so we have unified naming across whole feature. I understand cloned/copied is the same thing but it's not streamlined.